backend/benefit/applications/fixtures/application_basis.json

```diff
@@ -1,52 +1,2 @@
 [
-  {
```

Comment:
Was this left as an empty list intentionally or can this file be deleted?

Reply:
hmm actually it can be deleted, good point. Fixed.